internal/clients/fleet/fleetapi/doc.go

```diff
@@ -0,0 +1,5 @@
+package fleetapi
+
+//go:generate go run generate.go -o fleet-filtered.json
```

Comment:
Do you think we should use `-v v8.7.1` or some release tag instead of generating from `main` (by default)?

Reply:
Yeah I think we should once the API goes GA (or whenever we get the package policy changes?). I could quickly test it right now and if there's not any significant differences, we could lock to `v8.7.1` or something equivalent